hr_phone_validation/tests/test_phone_validation.py

```diff
@@ -0,0 +1,11 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestHrPhoneValidation(TransactionCase):
+
+    def test_phone_validation(self):
+        employee = self.env['hr.employee'].search([], limit=1)
+        phone = '+33612345678'
+        employee.write({'work_phone': phone, 'mobile_phone': phone})
+        employee._onchange_work_phone_validation()
```

Comment:
@fmdl nothing is tested here :smile:

Reply:
@simahawk I don't know why on test is OK and an other is not OK.

Reply:
@fmdl can you elaborate? :)

Reply:
![image](https://user-images.githubusercontent.com/16716992/40474456-5fa0ce30-5f3f-11e8-84cf-23d9b745262b.png)

Reply:
@fmdl sorry, missed that. Hmm the test seems to tell us the value has changed, is not equal anymore to what we expected: any other onchange/override/whatever that might change the value?
You should run the test locally precisely as travis does in that scenario to see what happens.

Reply:
Probably this test failure can be fixed by forcing the employee to have his own country_id.
Could you give it a try?